gulpTasks/definition.js

```diff
@@ -83,6 +84,7 @@ function externalDefs() {
     .pipe(replace(/undefined/g, 'any'))
     .pipe(replace(/Partial<[A-z]*>/g, 'any'))
     .pipe(replace(/const version: string;/g, ''))
+    .pipe(replace(/\s*implements any(?=\s)/gs, ''))
```

Comment:
Oh god, the fact that we have to support this file is so annoying 😂

Reply:
Agreed... Really glad we don't have to support older versions of Typescript in Headless